fem/bilinearform_ext.hpp

```diff
@@ -125,6 +125,15 @@ class FABilinearFormExtension : public EABilinearFormExtension
    FABilinearFormExtension(BilinearForm *form);
 
    void Assemble();
+   void RAP(OperatorHandle &A);
+   /** @note Always does `DIAG_ONE` policy to be consistent with
+       `ConstrainedOperator`. */
```

Comment:
I would suggesting adding a comment to `ConstrainedOperator` noting this limitation

Reply:
I meant adding a comment to `ConstrainedOperator::EliminateRHS` noting that what it does to the RHS is consistent with `DiagonalPolicy::ONE`.

Reply:
Done in [542bd7e](https://github.com/mfem/mfem/pull/3056/commits/542bd7ecd811751cb1d282873189c675aea392ce), tell me if this is what you suggested.

Reply:
Looks great, thanks